daemon/config_solaris.go

```diff
@@ -0,0 +1,65 @@
+package daemon
+
+import (
+	"github.com/docker/docker/opts"
+	flag "github.com/docker/docker/pkg/mflag"
+	"net"
+)
+
+var (
+	defaultPidFile = "/var/run/docker.pid"
+	defaultGraph   = "/var/lib/docker"
+	defaultExec    = "zones"
+)
+
+// Config defines the configuration of a docker daemon.
+// These are the configuration settings that you pass
+// to the docker daemon when you launch it with say: `docker -d -e lxc`
+type Config struct {
+	CommonConfig
+
+	// Fields below here are platform specific.
+	ExecRoot string `json:"exec-root,omitempty"`
+}
+
+// bridgeConfig stores all the bridge driver specific
+// configuration.
+type bridgeConfig struct {
```

Comment:
After you pointed this out I realized that some of the elements in bridgeConfig are yet to be evaluated for Solaris. Given this I've stubbed out bridgeConfig to exclude all platform specific elements for now. If bridgeConfig looks the same as now after they're added back I'll definitely create the bridge_config_unix.go file.